setup.py

```diff
@@ -13,10 +13,10 @@
 description = "CLI for Postgres Database. With auto-completion and syntax highlighting."
 
 install_requirements = [
-    "pgspecial>=1.11.5",
+    "pgspecial>=1.11.8",
     "click >= 4.1",
     "Pygments >= 2.0",  # Pygments has to be Capitalcased. WTF?
-    "prompt_toolkit>=2.0.6,<2.1.0",
+    "prompt_toolkit>=2.0.6,<3.0.0",
```

Comment:
Notice that this won't upgrade to version 3. it's ``<``, not ``<=``. Better use ``<3.1.0`` or ``4.0.0``.

Reply:
But that is intentional, given what Irina said, isn't it?

> it has incompatible changes according to @jonathanslenders, so I wanted to be on the safe side

Reply:
FWIW, I tried to execute the test suite using latest PT 3.0.2, and didn't notice any difference in the result, but I couldn't spend enough time to investigate on the failures (sorry about that)...
Given that nothing broke badly, neither in the tests, nor in my daily usage, hopefully the _incompatible changes_ are not impacting pgcli usage.

Reply:
To be clear, the same tests failed before and after upgrading PT.

Reply:
Yikes. We better make sure to fix whatever is broken then.